src/interfaces/I_InputProcessor.ts

```diff
@@ -0,0 +1,6 @@
+import { CommonInputModel } from "models/CommonInputModel";
+
+export interface I_InputProcessor {
```

Comment:
Cant really come up with a scenario where this would be necessary can you? 😄

Reply:
These extra 3 lines won't add extra bytes to package 😄 Interfaces will be also included in package (even if someone will use the sdk in JS), because we will expose the `.d.ts` files. I think that we should validate if custom processor implements `process` function. We can do it using `instanceof` or checking that function exists in prototype `{CustomProcessor}.prototype.process`. It's only a suggestion, if you wants tay with interface, good to me, but please change `I_InputProcessor` to `InputProcessor` :P

Reply:
Fair point, I am gonna change it :)

> @magicmatatjahu cant really come up with a scenario where this would be necessary can you? 😄 It reduces library size by using interfaces, when we dont have a scenario where this is relevant

Cant, have another class that is going to use the name `InputProcessor` in #5 which takes all our input processors and find out which one should be used 😄

Reply:
Done, changed the name to `AbstractInputProcessor` instead :)